crates/backend/src/codegen.rs

```diff
@@ -1024,114 +1024,115 @@ impl ToTokens for ast::ImportType {
 impl ToTokens for ast::ImportEnum {
     fn to_tokens(&self, tokens: &mut TokenStream) {
         let vis = &self.vis;
-        let name = &self.name;
-        let expect_string = format!("attempted to convert invalid {} into JSValue", name);
+        let enum_name = &self.name;
+        let name_str = enum_name.to_string();
+        let name_len = name_str.len() as u32;
+        let name_chars = name_str.chars().map(|c| c as u32);
```

Comment:
```suggestion
        let name_chars = name_str.chars().map(u32::from);
```

Reply:
done in 164c457b779941b9a211930b7976a3490dabb3ed